terraform/environments/data-platform/waf.tf

```diff
@@ -0,0 +1,87 @@
+resource "aws_wafv2_web_acl"  "waf" {
+  name  = "${local.environment}"
+  scope = "REGIONAL"
+
+  default_action {
+    allow {}
+  }
+
+  rule {
+    name     = "AWS-AWSManagedRulesCommonRuleSet"
+    priority = 1
+
+    override_action {
+      count {}
+    }
+
+    statement {
+      managed_rule_group_statement {
+        name        = "AWSManagedRulesCommonRuleSet"
+        vendor_name = "AWS"
+      }
+    }
+
+    visibility_config {
+        cloudwatch_metrics_enabled = false
```

Comment:
CloudWatch metrics might help to  gain visibility into how your WAF is performing and how it's affecting incoming web traffic. This information can be valuable for monitoring and optimizing your WAF configuration to better protect your web applications and respond to potential threats or issues.

Can we turn  it on ?